src/libs/colorpicker.c

```diff
@@ -363,6 +363,7 @@ static void _add_sample(GtkButton *widget, gpointer self)
   gtk_box_pack_start(GTK_BOX(data->samples_container), sample->container, TRUE, TRUE, 0);
 
   sample->color_patch = gtk_drawing_area_new();
+  gtk_widget_set_size_request(sample->color_patch, DT_PIXEL_APPLY_DPI(40), -1);
```

Comment:
Indeed this is not correct as we have made sure all hard coded size are removed. I'm not sure about the proper solution but IIRC it is not possible to set size of a  gtk_drawing_area_new() from CSS, you need to place it into a container (vbox for example) and you may need to name this widget to  add min-width/min-height in CSS.

Do you feel like trying that?

Reply:
No, I don't know how to make that. I have just tried to help as I've find where the issue was but I really don't know anything on coding except for CSS. Here I Just find that was this line.

Reply:
Ok, let me have a look then.